framework/contrib/ice_updater/include/LibcurlUtils.h

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
```

Comment:
I haven't looked but can we redistribute Curl sources ourselves?

On Tue, Sep 22, 2015 at 1:11 PM Jay Jay Billings <notifications@github.com>
wrote:

> In framework/contrib/ice_updater/include/LibcurlUtils.h
> <https://github.com/idaholab/moose/pull/5717#discussion_r40130429>:
>
> > @@ -0,0 +1,134 @@
> > +/*******************************************************************************
>
> +1 on its own directory. You may definitely have other uses for it. CURL
> is the lightest thing we could find, but I didn't do an exhaustive search.
> If you have suggestions, let us know and we will take a look.
> … <#msg-f:1513041797320435222_>
> On Tue, Sep 22, 2015 at 3:09 PM, Derek Gaston <notifications@github.com>
> wrote: In framework/contrib/ice_updater/include/LibcurlUtils.h <
> https://github.com/idaholab/moose/pull/5717#discussion_r40130157>: > @@
> -0,0 +1,134 @@ >
> +/*******************************************************************************
> I wonder if this stuff, that's actually agnostic of ICE, should go in its
> own contrib directory... we could use it in other places... — Reply to this
> email directly or view it on GitHub <
> https://github.com/idaholab/moose/pull/5717/files#r40130157>.
> -- Jay Jay Billings Oak Ridge National Laboratory Twitter Handle:
> @jayjaybillings <https://github.com/jayjaybillings>
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/idaholab/moose/pull/5717/files#r40130429>.
>